src/main/scala/no/ndla/searchapi/controller/SearchController.scala

```diff
@@ -332,6 +333,7 @@ trait SearchController {
       val relevances = paramAsListOfString(this.relevanceFilter.paramName)
       val contexts = paramAsListOfString(this.contextFilters.paramName)
       val grepCodes = paramAsListOfString(this.grepCodes.paramName)
+      val shouldScroll = paramOrNone(this.scrollId.paramName).exists(InitialScrollContextKeywords.contains)
```

Comment:
Antar at `None.exists()` returnerer false?

Reply:
Yes, om du lurer på noe funksjonalitet i standardbiblioteket så er ofte scaladoc'en ganske rå:
```
  /** Returns true if this option is nonempty '''and''' the predicate
   * $p returns true when applied to this $option's value.
   * Otherwise, returns false.
   *
   * This is equivalent to:
   * {{{
   * option match {
   *   case Some(x) => p(x)
   *   case None    => false
   * }
   * }}}
   *  @param  p   the predicate to test
   */
  @inline final def exists(p: A => Boolean): Boolean =
```